esp/smc/SMCLib/TpContainer.cpp

```diff
@@ -802,3 +802,13 @@ StringArray & getRoxieDirectAccessPlanes(StringArray & planes, StringBuffer &def
 
     return planes;
 }
+
+extern TPWRAPPER_API SecAccessFlags getDropZoneScopePermissions(IEspContext & context, const char * dropZoneName, const char * dropZoneServer, const char * dropZonePath)
```

Comment:
it would be clearer to remove the out of context dropZoneServer parameter here,
and in the non-containerized version the dropZoneName, and unify into a single parameter.

Reply:
Is this related to: https://github.com/hpcc-systems/HPCC-Platform/pull/16710#discussion_r1029416424 ?

Reply:
Yes